src/services/activities/activities.types.ts

```diff
@@ -0,0 +1,28 @@
+export type ActivityType = 'create' | 'update' | 'delete';
+export type ActivityTargetType = 'person' | 'asset' | 'tenure';
+
+interface PersonName {
```

Comment:
The Data here is not limited to PersonName. It can be any field on the Person Entity. I would do a `Partial<Person>`, or something similar.

Reply:
updated to `Partial<Person>` for the timebeing.
this will likely change in the next PR for the next ticket